docs/extensibility/theme-support.md

```diff
@@ -9,10 +9,22 @@ To opt-in for one of these features, call `add_theme_support` in the `functions.
 ```php
 function mytheme_setup_theme_supported_features() {
 	add_theme_support( 'editor-color-palette',
-		'#a156b4',
-		'#d0a5db',
-		'#eee',
-		'#444'
+		array(
+			'name' => 'strong magenta',
```

Comment:
Why not use the `name` as key and the `color` as value?

```php
array(
    'strong magenta' => '#a156b4',
)
```
?

Reply:
Because we want themes to be able to set the order in which colors appear in the palette, and when this gets converted to a JS object we would not have the guarantee that the order would be kept as the object is an unordered collection of properties.

Reply:
Can we map it to an ordered array before it gets exposed to JavaScript?

PHP doesn't care as much about the order when strings are involved.